internetarchive/openlibrary
# fix gitpod covers

Closes https://github.com/internetarchive/openlibrary/issues/5834




### Technical

A pretty simple fix at the end of the day, just injecting the right domain in to the config in the gitpod setup

### Testing

1. Start a gitpod of this PR at https://gitpod.io/#https://github.com/internetarchive/openlibrary/pull/9869
2. Upload an image
3. See the image works

### Screenshot


### Stakeholders
